engine/message/queue.go

```diff
@@ -23,6 +19,36 @@ var typePriority = map[Type]int{
 	TypeAlertStatusUpdateBundle: 4,
 }
 
+var perCMThrottle = ThrottleConfig{
```

Comment:
Maybe we should move these into `throttle.go` or `ratelimits.go` for visibility.